sql_maintenance.py

```diff
@@ -0,0 +1,69 @@
+import os
+import subprocess
+
+# Define your SQL Server connection details
+server = os.environ['DB_CONNECTION_URL']
+database = os.environ['my_db_list']
+username = os.environ['DB_USERNAME']
+password = os.environ['DB_PASSWORD']
+
+sql_script = "/app/query.sql"
+
+#converting in to absolute path
+sql_script = os.path.abspath(sql_script)
+
+query_to_run = ["exec [dbo].[AzureSQLMaintenance] @Operation='index',@mode='smart',@LogToTable=1", "exec [dbo].[AzureSQLMaintenance] @Operation='statistics',@mode='dummy' ,@LogToTable=1"]
```

Comment:
Works like this but maybe better to have three steps like in automation for better debugging/logging afterwards.

![image](https://github.com/uniper/uniper-azure-db-maintenance/assets/43784665/0382f80f-cef0-48bd-9363-42a8e4e2bbc5)

It would be easier to add appropriate logging on explicit calls.